[Task]: Fixes Full page cache documentation and adds an example with DependencyInjection (#14435)
pimcore/pimcore: doc/Development_Documentation/19_Development_Tools_and_Details/09_Cache/03_Full_Page_Cache.md

| Exclude Patterns | You can define some exclude patterns where the cache doesn't affect. The patterns have to be valid regular expressions (including delimiters) and different patterns should be seperated by `,` |
| Disable Cookie | You can define an additional cookie-name which disables the cache. The cookie "pimcore_admin_sid" (used for the Pimcore admin UI) ALWAYS disables the output-cache to make editor's life easier ;-) 

## Disable the Full Page Cache in your Code
Sometimes it is more useful to deactivate the full page cache directly in the code, for example when
it's not possible to define an exclude-regex, or for similar reasons.

In this case you can use obtain the pull page cache service from the container and disable it, eg. in an Controller via DI:
```php
<?php

    use Pimcore\Bundle\CoreBundle\EventListener\Frontend\FullPageCacheListener;
    
    public function portalAction(Request $request, FullPageCacheListener $fullPageCacheListener)
    {

>        $FullPageCacheListener->disable("Your disable reason");

Reviewer: ```suggestion
       $fullPageCacheListener->disable('Your disable reason');
```

Author: sorry missed that and when i wanted to change that there some ühone call coming in :) -> fixed

Reviewer: No problem :)

| ------ | ----------- |
| Enable | Set to true to enable to full page cache. |
| Lifetime | You can optionally define a lifetime (in seconds) for the  full page cache. If you don't do, the cache is evicted automatically when there is a modification in the Pimcore Backend UI. If there is a lifetime the item stays in the cache even when it is changed until the TTL is over. The lifetime is useful if you have embedded some items which are not directly in the cms, like rss feeds, or twitter messages over the API. It is also highly recommended to specify a lifetime on high traffic websites so that the frontend (caches) isn't affected by changes in the admin-UI. Otherwise on every change in the admin-UI the whole output-cache is flushed, what can have drastic effects to the server environment. |

> | Exclude Patterns | You can define some exclude patterns where the cache doesn't affect. The patterns have to be valid regular expressions (including delimiters). seperate patterns with , |

Reviewer 2: ```suggestion
| Exclude Patterns | You can define some exclude patterns where the cache doesn't affect. The patterns have to be valid regular expressions (including delimiters). seperate patterns with `,` |
```

Reviewer 3: The word after the dot (end of phrase) looks grammarly incorrect, what about
```suggestion
| Exclude Patterns | You can define some exclude patterns where the cache doesn't affect. The patterns have to be valid regular expressions (including delimiters) and different patterns should be seperated by `,` |
```

Reviewer 3: But i have an input: should be changing the config node to work as arrayNode instead (and be per line like it meant to be in the docs) and BC break in 11?
~What if the regex pattern has a comma ~a range like `{1,4}`(range works with -)~, the explode could break by this comma, isn't it?~ nevermind, commas should be url encoded as `%2C`

https://github.com/pimcore/pimcore/blob/b1bab678a43a82e500c05631bc2c2be192be85c5/bundles/CoreBundle/src/EventListener/Frontend/FullPageCacheListener.php#L184

```php
<?php
$this->get(\Pimcore\Bundle\CoreBundle\EventListener\Frontend\FullPageCacheListener::class)->disable("Your disable reason");

> or better with DI 

Reviewer 4: I would replace `$this->get` example with DI as this anyway wont work with v11. see https://github.com/pimcore/pimcore/pull/12048

Author: yes totally right! 
Done
